wave_vision/tests/descriptor_tests/brisk_tests.cpp

```diff
@@ -0,0 +1,197 @@
+// C++ Headers
+#include <algorithm>
+
+// Libwave Headers
+#include "wave/wave_test.hpp"
+#include "wave/vision/descriptor/brisk_descriptor.hpp"
+#include "wave/vision/detector/fast_detector.hpp"
+
+/** The wave namespace */
```

Comment:
Hey, "The wave namespace" is already implied by `namespace wave`
Doxygen only uses one description anyway, should avoid repeating it.